content/docs-v1/multiformat-output.md

```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
 
 ## Site Output
 
-Create the HTML files for your project by running `quire site` in your command-line shell. The files will be built into your project’s `site` folder along with all the necessary static assets like image files, stylesheets and script files. The `site` file will be updated and overwritten each time you run `quire site`.
+Create the HTML files for your project by running `quire build` in your command-line shell. The files will be built into your project’s `_site` folder along with all the necessary static assets like image files, stylesheets and script files. The `_site` file will be updated and overwritten each time you run `quire build`.
 
-{{< q-class "box tip" >}}
-- If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire site`.
-{{< /q-class >}}
-
-You can hide specific pages of your project from the site output by adding `online: false` to the page YAML. Or conversely, you can add pages exclusively to the site output, by adding `epub: false` and `pdf: false` to the page YAML and leaving the `online` attribute unset, or set to `true`.
+If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire build`.
 
-Unlike `epub: false` and `pdf: false`, adding `online: false` does not stop a page from being built and included in the site output. Rather, it just unlinks those pages and adds a metadata tag to them to prevent them from being indexed by search engines. You may want to manually delete the pages from the `site` folder after outputting and before uploading to a web server if you want to be fully certain they can’t be accessed online under any circumstances.
+Links to download the EPUB, MOBI, and PDF versions are automatically included in the sidebar menu of the online version of your project. These links can be removed or modified by making changes to the `resource_link` information in your `content/_data/publication.yaml` file. You can add links to these files from *anywhere* in your markdown files by linking to `_assets/downloads/output.epub`, `_assets/downloads/output.mobi`, and `_assets/downloads/output.pdf` respectively.
```

Comment:
I'm not clear on what you mean by "You can add links to these files from *anywhere* in your markdown files by linking to `_assets/downloads/output.epub`, `_assets/downloads/output.mobi`, and `_assets/downloads/output.pdf` respectively".

Reply:
I guess I meant that you could have a sentence on a markdown page like "Download the free PDF and EPUB versions of this book.". I can clarify that in the write-up, or should I just delete that part?